pydjamodb/test_runner.py

```diff
@@ -4,17 +4,16 @@
 from django.db import connections
 from django.test.runner import ParallelTestSuite, DiscoverRunner
 
+from germanium.signals import set_up, tear_down
```

Comment:
You should add germanium to setup.py as it is now required for the library. (not just for the failing tests, although the test app will need a germanium version bump or removal of the dependency as this app will have it on its own)

Reply:
no it is not required

use test runner is not mandatory with using pydjamodb

Reply:
added try except too support test runner without germanium